functions/exercise11/scriptSpec.js

```diff
@@ -0,0 +1,9 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getSecondLowestAndSecondGreatestNumbers*/
+
+describe("Getting second lovest and second gratest numbers with array", function () {
+    it("[1, 2, 3, 4, 5] is 2 and 4 ", function () {
```

Comment:
I do not undestand the description here. Please write it in the way it is understandable. What is 2 and what is 4 in this context?
Why do you have a space after 4?